maestro/financial-connections/Testmode-PaymentIntent-TestInstitution-Networking.yaml

```diff
@@ -6,7 +6,7 @@ tags:
 ---
 - startRecording: ${'/tmp/test_results/testmode-paymentintent-testinstitution-networking-' + new Date().getTime()}
 - clearState
-- openLink: stripeconnectionsexample://playground?flow=PaymentIntent&financial_connections_override_native=native&merchant=networking_testmode
```

Comment:
is this intended? seems unrelated to example app

Reply:
it is! added permissions to the deep link. Maestro opens the example app via deeplink, and wanted to add these permissions for the networking flow.